.github/workflows/tests.yml

```diff
@@ -0,0 +1,138 @@
+name: histolab
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install OS Dependencies
+        run:   |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+               sudo apt install openslide-tools
+          elif [ "$RUNNER_OS" == "macOS" ]; then
+               brew install openslide
+          else
+               echo "$RUNNER_OS not supported"
+               exit 1
+          fi
+      - name: Install dependencies
+        run: |
+            python -m pip install --upgrade pip
+            python -m pip install -e .[testing]
+            python -m pip install flake8
+            python -m pip install pooch
+      - name: Lint with flake8
+        run: |
+            # stop the build if there are Python syntax errors or undefined names
+            flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+            # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+            flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --statistics
```

Comment:
why calling flake8 twice with different parameters? what kind of behavior do you want here? 
moreover, what happens if we change the `.flake8`  conf file?